roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -1,13 +1,13 @@
 ---
 - name: Create passthrough route for docker-registry
-  command: >
-    {{ openshift.common.client_binary }} create route passthrough
-    --service docker-registry
-    --config={{ openshift_hosted_kubeconfig }}
-    -n default
-  register: create_docker_registry_route
-  changed_when: "'already exists' not in create_docker_registry_route.stderr"
-  failed_when: "'already exists' not in create_docker_registry_route.stderr and create_docker_registry_route.rc != 0"
+  oc_route:
+    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
```

Comment:
This one I think is fine. But I need to reintroduce the copy in the cockpit-ui module as there is one command that doesn't use a module yet and needs a kubeconfig. Using the same kubeconfig could modify it and cause problems downline. I'll comment around it so it's obvious why the copy is occuring.